packages/es-components/src/index.js

```diff
@@ -10,6 +10,10 @@ export Menu from './components/containers/menu/Menu';
 export TabPanel from './components/containers/tabPanels/TabPanel';
 export StripedContainer from './components/containers/striped-container/StripedContainer';
 export CheckboxLabel from './components/containers/checkboxLabel/CheckboxLabel';
+export Table from './components/containers/Table';
```

Comment:
These imports need to be from `./components/containers/table/Table`

Reply:
I've got it taken care of on my end. 👍

Reply:
What do you think about namespacing the sub elements so it's `Table.Row`, `Table.Cell`, etc.? That way you just have the one default Table export as well.

Reply:
I didn't think about it at the time. I will do that for the `next` release. 👍